index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+
+	<article class="tops">
+		<header id="header" class="ones"><h1>Walri</h1></header>
+		<div>
+		<div><img src="images/walri.jpg" alt="a single walrus" height="168px" width="188px"></div>
+		<div><p class="words">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```

Comment:
Requirement 4 "Each product should have three sections."
In actuality your tags are divided into divs, but I think something that would make it more clear were if you used 'section' tags instead.